recipe/meta.yaml

```diff
@@ -29,11 +29,8 @@ requirements:
     - gshhg-gmt
     - dcw-gmt
   run:
-    - libblas
-    - libcblas
-    - liblapack
```

Comment:
Removing these from runtime, based on https://conda-forge.org/docs/maintainer/knowledge_base/#blas saying that these should only be needed on host?